api/base/serializers.py

```diff
@@ -88,6 +89,26 @@ def to_internal_value(self, data):
         return super(JSONAPIListField, self).to_internal_value(data)
 
 
+class AuthorizedCharField(ser.CharField):
```

Comment:
I'm fine with that. 

Another simple alternative is to add a writable method field:

```python
class WritableMethodField(SerializerMethodField):
    def __init__(self, method_name=None, write_method_name=None, **kwargs):
        assert write_method_name is not None, 'must provide write_method_name'
        self.write_method_name = write_method_name
        super(SerializerMethodField, self).__init__(method_name=method_name, **kwargs)

    def to_internal_value(self, data):
        method = getattr(self.parent, self.write_method_name)
        return method(value)
```

```python
    content = WritableMethodField(write_method_name='write_content')
```

If we're going to add a new field type in this PR anyway, perhaps we go with this for now? If this will add significant work, then we can defer.